Earthquakes/Earthquakes/Earthquake.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Earthquake.swift
+//  Earthquakes
+//
+//  Created by Pratikbhai Patel on 10/25/16.
+//  Copyright © 2016 Pratik Patel. All rights reserved.
+//
+
+import Foundation
+
+class Earthquake {
+    
+    let magnitude: Double = 0
+    let time: Date = Date()
+    let url: URL? = nil
+    let place: String = ""
+    let detail: URL? = nil
```

Comment:
Do we really need this to be optional? Why can't we initialize with empty URL and assign later?